src/components/Table/Table.tsx

```diff
@@ -46,7 +46,7 @@ const Container = withStyles({
     },
   },
   // This is the default Container, only styles above have changed.
```

Comment:
Will you remove this comment now that it's incorrect?

Reply:
Well, I'll update it 😉